lib/trento/activity_logging/parser/phoenix_conn_parser.ex

```diff
@@ -108,6 +108,25 @@ defmodule Trento.ActivityLog.Logger.Parser.PhoenixConnParser do
     |> redact(:password_confirmation)
   end
 
+  def get_activity_metadata(
+        action,
+        %Plug.Conn{
+          params: params
+        }
+      )
+      when action in [
+             :cluster_checks_execution_request,
+             :host_checks_execution_request
+           ] do
+    action
+    |> case do
+      :cluster_checks_execution_request -> {:cluster_id, Map.get(params, :cluster_id)}
+      :host_checks_execution_request -> {:host_id, Map.get(params, :id)}
+    end
+    |> List.wrap()
+    |> Map.new()
+  end
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏
```suggestion
  def get_activity_metadata(
        :cluster_checks_execution_request,
        %Plug.Conn{
          params: params
        }
      ),
      do: %{cluster_id: Map.get(params, :cluster_id)}

  def get_activity_metadata(
        :host_checks_execution_request,
        %Plug.Conn{
          params: params
        }
      ),
      do: %{host_id: Map.get(params, :id)}
```

Reply:
Yes :heart: